src/cloud/project/project-patch.md

```diff
@@ -5,26 +5,20 @@ functional_areas:
   - Cloud
   - Upgrade
 ---
-The [{{site.data.var.mcp-prod}}](https://github.com/magento/magento-cloud-patches) package
-delivers Magento patches, which improve the integration of all `{{site.data.var.ee}}` versions with Cloud environments and supports quick delivery of critical and custom fixes.
 
-The {{ site.data.var.mcp }} package is a dependency for the {{site.data.var.ct}} package and is installed or updated when you install or update the {{ site.data.var.ct }} package version. You can also use and manage the {{ site.data.var.mcp-prod }} as a stand-alone package for an existing {{ site.data.var.ee }} project.
+The [Magento Quality Patches](https://github.com/magento/quality-patches) and [{{ site.data.var.mcp-prod }}](https://github.com/magento/magento-cloud-patches) packages deliver Magento [individual patches]({{ site.baseurl }}/release/policy/#individual-patch), which improve the integration of all {{site.data.var.ee}} versions with Cloud environments and supports quick delivery of critical and custom fixes. You can use these packages to apply, revert, and view general information about all individual patches that are available for Magento.
```

Comment:
You are welcome to use reference links, if you like. Most Cloud topics are trending that way.

Reply:
I'm actually not a fan of reference links. For me, they actually make link maintenance more burdensome.